templates/ceph/managed.html

```diff
@@ -0,0 +1,256 @@
+{% extends "kubernetes/base_kubernetes.html" %}
+
+{% block title %}Managed Ceph | Ceph{% endblock %}
+
+{% block meta_description %}Canonical is the leading managed ceph provider, we build, support and manage ceph clusters in any location. {% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1Qv379nYnod9e4jMrsdDND8D7mbPyqELi2lowjkfD-gU{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-bottomed">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-7">
+      <h1>Fully managed Ceph</h1>
+      <p>Reduce operational complexity by allowing Canonical’s experts to operate and maintain your Ceph cluster.</p>
+      <ul class="p-list">
+				<li class="p-list__item is-ticked">Fully managed on your servers, in your own data center, or in a co-location facility</li>
+				<li class="p-list__item is-ticked">Built using a proven reference architecture</li>
+				<li class="p-list__item is-ticked">Customise the cluster architecture based on your needs</li>
+				<li class="p-list__item is-ticked">Integrated with your infrastructure, on-prem, hybrid, or public cloud</li>
+				<li class="p-list__item is-ticked">Existing workload analysis and migration services</li>
+				<li class="p-list__item is-ticked">Enterprise SLA for all protocols</li>
+				<li class="p-list__item is-ticked">Your team retains access to all machines at all times</li>
+				<li class="p-list__item is-ticked">Modern monitoring and log management</li>
+				<li class="p-list__item is-ticked">We will hand over the keys when you are ready</li>
+      </ul>
+      <a href="/ceph/contact-us" class="p-button--positive js-invoke-modal" >Get in touch</a>
+			<p>
+				<a href="https://www.brighttalk.com/webcast/6793/428078h" class="p-link--external">Watch the webinar - Redefine your enterprise storage with Ceph</a>
+			</p>
+    </div>
+    <div class="col-5 u-align--center u-hide--small">
+			{{ image (
+				url="https://assets.ubuntu.com/v1/e0461e14-ceph-dark.svg",
+				alt="Ceph",
+				width="270",
+				height="110",
+				hi_def=True,
+				loading="auto"
+				) | safe
+			}}
+    </div>
+  </div>
+</section>
+<section class="p-strip strip-light">
+  <div class="u-fixed-width">
+    <h2>Build, operate, transfer and support</h2>
+    <p class="u-sv3">Deploy a fully managed Ceph cluster in a few weeks with these simple steps.</p>
+  </div>
+  <div class="u-fixed-width">
+    <ol class="p-stepped-list">
+      <li class="p-stepped-list__item">
+        <div class="row u-equal-height">
+          <div class="col-6">
+            <h3 class="p-stepped-list__title"> Design </h3>
+						<p>Our experts work with your team to <a href="/ceph/consulting" class=>design a Ceph cluster</a> suited to your needs.  We guide you through hardware and configuration decisions, to ensure that the resulting cluster is tailored to meet your performance, capacity, security and integration requirements.</p>
```

Comment:
Make sure you merge the other [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/10354) first as this links to it